pkgs/by-name/di/distrobox-tui/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "distrobox-tui";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "phanirithvij";
+    repo = pname;
```

Comment:
Use the literal string instead of pname to allow pname override.

Reply:
done